moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -109,7 +109,8 @@ bool MoveItCpp::loadPlanningSceneMonitor(const PlanningSceneMonitorOptions& opti
   {
     // Start state and scene monitors
     RCLCPP_INFO(LOGGER, "Listening to '%s' for joint states", options.joint_state_topic.c_str());
-    planning_scene_monitor_->startStateMonitor(options.joint_state_topic, options.attached_collision_object_topic);
+    planning_scene_monitor_->startStateMonitor(rclcpp::Node::SharedPtr(), options.joint_state_topic,
```

Comment:
planning_scene_monitor_ was built with the `move_group` node, if you pass in something invalid it will use the node it was constructed with. also see https://github.com/ros-planning/moveit2/pull/217#discussion_r471973610